packages/editor/src/components/post-featured-image/style.scss

```diff
@@ -31,8 +30,9 @@
 	@include reduce-motion("transition");
 	box-shadow: 0 0 0 0 var(--wp-admin-theme-color);
 	overflow: hidden; // Ensure the focus style properly encapsulates the image.
-	outline: $border-width solid rgba(0, 0, 0, 0.1);
 	outline-offset: -#{$border-width};
+	min-height: $grid-unit-50 * 2;
+	margin-bottom: $grid-unit-10;
```

Comment:
If we use `$grid-unit-20`, the space between this and each other item in the summary is the same: 

![CleanShot 2024-03-22 at 10 43 56](https://github.com/WordPress/gutenberg/assets/1813435/8b490d09-cc70-43f4-b73c-71c08c9c0a6a)

Reply:
Nice!